cypress.config.js

```diff
@@ -2,6 +2,9 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/login',
+    viewportWidth: 1100,
+    viewportHeight: 1300,
     setupNodeEvents(on, config) {
```

Comment:
The 'setupNodeEvents' function is empty and might not be needed in this case. Consider removing it if it's not being used. 🧹